extensions/data-transfer/portability-data-transfer-flickr/src/test/java/org/datatransferproject/datatransfer/flickr/FlickrTestUtils.java

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package org.datatransferproject.datatransfer.flickr.photos;
+package org.datatransferproject.datatransfer.flickr;
 
 import com.flickr4java.flickr.photos.Photo;
 import com.flickr4java.flickr.photos.Size;
 import com.flickr4java.flickr.photosets.Photoset;
 import java.util.Collections;
 import java.util.UUID;
 
-class FlickrTestUtils {
+public class FlickrTestUtils {
```

Comment:
are these PhotoSet things related to videos too? (like is "photoset" just flickr langauge for "album" or something?). No need to rename everything/anything - but maybe just a comment on this class would be helpful.